src/infra/HttpRequester.ts

```diff
@@ -1,15 +1,17 @@
-import { HttpRequesterService } from '../services/HttpRequesterService';
-import { AnalyticsPayload } from '../models/AnalyticsPayload';
+import { HttpRequesterService } from '../services';
+import { AnalyticsPayload } from '../models';
 import fetch from 'cross-fetch';
 
 /**
  * Responsible for making web requests.
+ *
+ * @public
```

Comment:
Does this need to be annotated as `public`? Ideally it should just be an internal implementation detail.

Reply:
So i did this because there was a warning in the auto-generated markdown file bc this is part of the method signature of the AnalyticsReporters

Reply:
the warning is:
```
// Warning: (ae-forgotten-export) The symbol "HttpRequesterService" needs to be exported by the entry point index.d.ts
```

Reply:
oh nevermind i see why it's not warning on the SearchAnalyticsReporter

Reply:
Why do the various reporters need to be exported? We treated the `AnalyticsReporter` as an implementation detail, IIRC. We did not export it, users could not instantiate it manually. They had to use the `provide` method.

Reply:
let me look back at master, i thought I was following what it was doing but I may have over-exported things